app.json

```diff
@@ -1,29 +1,62 @@
 {
   "name": "retool",
-  "description": "Retool heroku",
+  "description": "Retool Heroku",
+  "keywords": [
+    "retool"
+  ],
+  "logo": "https://avatars.githubusercontent.com/u/33817679",
+  "repository": "https://github.com/tryretool/retool-onpremise",
   "stack": "container",
+  "formation": {
+    "web": {
+      "quantity": 1,
+      "size": "performance-m"
+    }
+  },
   "env": {
-    "NODE_ENV": {
-      "description": "The environment to run Retool",
-      "value": "production"
+    "ENCRYPTION_KEY": {
+      "description": "A secret key for encrypting secret credentials when connecting Retool to other databases or API endpoints",
+      "required": true,
+      "generator": "secret"
+    },
+    "HEROKU_HOSTED": {
+      "required": true,
+      "value": "true"
     },
     "HOST_ENV": {
```

Comment:
as far as I can tell there's no product use of HOST_ENV or HEROKU_HOSTED, so we should be able to remove these variables from the template